backend/src/api/routes/ai/index.routes.ts

```
@@ -241,6 +241,21 @@ function rotateProviderApiKey(provider: AIProvider, openRouterProvider: OpenRout
   }
 }
 
+function handleRouteError(error: unknown,next: NextFunction,fallbackMessage: string): void {
```

ESLint

[error] Replace `next:·NextFunction,` with `·next:·NextFunction,·`

(prettier/prettier)